frontend/fingertips-frontend/lib/search/searchService.ts

```diff
@@ -0,0 +1,62 @@
+import { IndicatorSearch, IndicatorSearchResult } from './searchResultData';
+import { AzureKeyCredential, SearchClient } from '@azure/search-documents';
+
+type Indicator = {
+  IID: string;
+  Descriptive: {
+    Name: string;
+    Definition: string;
+    DataSource: string;
+  };
+  DataChange: {
+    LastUploadedAt: string;
+  };
+};
+
+const getEnvironmentVariable = (name: string): string => {
+  return (
+    process.env[name] ??
+    (() => {
+      throw new Error(`Missing environment variable ${name}`);
+    })()
+  );
+};
```

Comment:
This might make more sense in its own file, as I expect we'll want to use it from other places.